src/components/project/workflow/transitions/conditions.ts

```diff
@@ -23,6 +24,45 @@ export const IsMultiplication: Condition = {
   },
 };
 
+export const hasValidRoleForProjectType: Condition = {
```

Comment:
This is an authorization concern, so the policies should handle it. We need to just figure out how to get the "internship type condition" there. Let me think about it.
I get that's it's kinda weird we have two ways and both have conditions.